compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -51,29 +53,44 @@ def __executor_atexit():
 threading.Thread(target=__executor_atexit).start()
 
 
-class TaskSubmissionInfo:
+UUID_LIKE_T = t.Union[uuid.UUID, str]
+
```

Comment:
Ooh, fun addition!  I wonder if there's a more general and/or appropriate spot for this in the SDK?  We could (should?) certainly use this in a number of places.  (For example, my comment about typing in the signature of `Client.create_batch()`.)

Reply:
Moved to a `utils` module.